pkg/exec/exec.go

```diff
@@ -14,66 +14,76 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package exec contains helpers for os/exec
+// Package exec contains an interface for executing commands, along with helpers
+// TODO(bentheelder): add standardized timeout functionality & a default timeout
```

Comment:
this is a huge follow up I intend to do, reworking this so that Run() respects a standardized timeout whether running commands locally or on the nodes, commands hang sometimes, particularly when we have bugs, we should ensure that no command can hang indefinitely.

Reply:
I think handling this properly is pretty key to robustness